.github/workflows/build.yml

```diff
@@ -20,7 +21,10 @@ jobs:
           cache: "npm"
 
       - name: Install JS dependencies
-        run: npm ci
+        run: npm install
```

Comment:
`npm install` can deviate from the versions in `package-lock.json`, whereas `npm ci` will fail if versions can't match exactly. most of the time, you want `npm ci` in your build so you know exactly what versions made it into production.

however, this is a package used by multiple projects. `npm install` better simulates the real world usage and may surface different, more useful errors.